### PR TITLE
Cleanup project template ide.host.json files

### LINKED ISSUE #41631: Visual Studio does not retain 'Do not use top-level statements' check.
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

First of all, thank you ASP.NET Core Team for adding an option in templates to avoid top-level statement. 

However, we have noticed that  Visual Studio is not retaining the `Do not use top-level statement` check.

First Time:

Capture1

Second Time:

Capture2

Note: All other checks are retaining their last choice.

### Expected Behavior

Check should retain the last choice.

### Steps To Reproduce

_No response_

### Exceptions (if any)

_No response_

### .NET Version

6.0.300 and 7.0.100-preview.4.22252.9

### Anything else?

Visual Studio Version: 2022 17.3 preview 1.

### LINKED ISSUE #41637: Consider setting PersistenceScope for other template options
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Similar to #41631, we should consider adding PersistenceScope for the other options which we want to persist, such as UseMinimalAPIs or OpenAPI.

//cc @DamianEdwards 

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Exceptions (if any)

_No response_

### .NET Version

_No response_

### Anything else?

_No response_

### LINKED ISSUE #41638: Cleanup ide.host.json files.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Consider the following changes:

1. New schema is available on schemastore: ide.host instead of vs-2017.3.host
2. Remove ports section - not used by current Visual Studio
3. Remove supportedAuthentication section - not used by current Visual Studio

Anything else that is not documented in the schema, is potential removal candidate, please confirm with @phenning.

### Expected Behavior

Clean up of unreferenced properties, updated schema.

### Steps To Reproduce

_No response_

### Exceptions (if any)

_No response_

### .NET Version

_No response_

### Anything else?

Visual Studio 2022 17.3 Preview 1

### PR DESCRIPTION
**Pull request by DamianEdwards**
# Cleanup project template ide.host.json files

- Make option values sticky in VS
- Remove unused sections in ide.host.json
- Fix template local tests scripts broken from test project rename

Fixes #41638 #41637 #41631

### CODE REVIEW

**Review by phenning — APPROVED** [review]

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/ide.host.json [hunk 1]
```diff
@@ -1,50 +1,24 @@
...
-      "useHttps": true
-    }
-  ],
   "disableHttpsSymbol": "NoHttps",
   "symbolInfo": [
     {
       "id": "Hosted",
-      "isVisible": "true"
+      "isVisible": true,
+      "PersistenceScope": "TemplateGroup"
```
Comment (phenning) [inline src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/ide.host.json:10]: nit: schema actually has scope as camel cased.

Comment (DamianEdwards) [reply src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/ide.host.json:10]:
```suggestion
      "persistenceScope": "templateGroup"
```

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/ide.host.json [hunk 2]
```diff
@@ -1,50 +1,24 @@
...
       "id": "Hosted",
-      "isVisible": "true"
+      "isVisible": true,
+      "persistenceScope": "templateGroup"
     },
     {
       "id": "PWA",
-      "isVisible": "true"
+      "isVisible": true,
+      "PersistenceScope": "TemplateGroup"
```
Comment (DamianEdwards) [inline src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/ide.host.json:15]:
```suggestion
      "persistenceScope": "templateGroup"
```

**Pull request merged**